### DISCUSSION TITLE
The first implementation of bark voice cloning I've found that actually works.

### DISCUSSION THREAD

**Discussion by remghoost** [gitmylo/audio-webui#31] [General]
Training is straight forward.
Audio generation is straight forward.

Tested a model with 7 seconds of audio, trained at 160 epochs so far and it sounds _surprisingly close_ so far.
The voice I'm cloning has a very specific British accent and I can already hear their canter/accent after that many epochs.
Have to mess around a bit more with it to really get a feel for if this works as well as I'm thinking it does.

Have only used it for a day or so, but this seems to be exactly what I've been looking for for about 6 months or so.
Extremely solid amount of API endpoints as well (though I haven't used them _yet_)

Just wanted to leave this here for anyone else that's looking. 
There are dozens of us, I tell you! Dozens!

**Comment by gitmylo** [comment]
you trained an rvc model for 160 epochs on only 7 seconds of audio and it worked pretty well? That's amazing. You can also train in way more audio, like half an hour for example, that's usually pretty good.
